concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -69,20 +72,37 @@ def report_computed_task(timestamp = None, task_to_compute = None):
     with freeze_time(timestamp):
         return message.tasks.ReportComputedTask(
             task_to_compute = task_to_compute,
-            size=10,
+            size=REPORT_COMPUTED_TASK_SIZE,
         )
 
 
+def calculate_timestamp(current_time, cluster_consts):
```

Comment:
acatually, you need just `concent_messaging_time`, please change function signature (for all three functions)

Reply:
:+1: done